var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -28,6 +28,7 @@ class Mesa(MesonPackage):
     version('19.0.2', tag='mesa-19.0.2')
     version('19.0.1', tag='mesa-19.0.1')
     version('19.0.0', tag='mesa-19.0.0')
+    version('17.2.3', tag='mesa-17.2.3')
```

Comment:
Have we verified that this version of Mesa builds with this recipe? I'm not sure which versions build with which build stem (meson vs autotools).

Reply:
No, but the buildcache creation fails if it is missing because the version is an entry in packages.yaml.

Reply:
I think the error message was that it couldn't extrapolate the source url from the version number.